gensim/summarization/bm25.py

```diff
@@ -66,14 +62,56 @@ class BM25(object):
         List of document lengths.
     """
 
-    def __init__(self, corpus):
+    def __init__(self, corpus, k1=1.5, b=0.75, epsilon=0.25):
         """
+
+        'k1 is a variable which helps determine term frequency saturation characteristics.
```

Comment:
First of all, the documentation is going to be displayed [on the Gensim website](https://radimrehurek.com/gensim/summarization/bm25.html). Therefore, you should use the [reStructuredText markup](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) for links, inline code, and citations.

Second of all, the documentation should follow the [NumPy docstring standard](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard). The class docstring should begin with a one-line summary, followed by one or more paragraphs of class description. Discussion of k1 and b should definitely be abbreviated and perhaps moved to the corresponding parameter descriptions. Multiple paragraphs are supported in parameter descriptions, but shorter is often better.